iocore/net/SSLNetVConnection.cc

```diff
@@ -303,6 +310,10 @@ ssl_read_from_net(SSLNetVConnection *sslvc, EThread *lthread, int64_t &ret)
     ret = bytes_read;
 
     event = (s->vio.ntodo() <= 0) ? SSL_READ_COMPLETE : SSL_READ_READY;
+    if (sslErr == SSL_ERROR_NONE && s->vio.ntodo() > 0) {
+      // We stopped with data on the wire (to avoid overbuffering).  Make sure we are triggered
+      sslvc->read.triggered = 1;
```

Comment:
The `triggered` is set in `NetHandler::mainNetEvent()`.
There is no one to clear `triggered` in `ssl_read_from_net()`.
It is only cleared in `SSLNetVC::net_read_io()` if `SSL_READ_WOULD_BLOCK` or `SSL_READ_EOS` or `SSL_READ_ERROR` is returned from `ssl_read_from_net()`.

In another word, the `read.triggered` is always TRUE here, you can try `ink_assert(sslvc->read.triggered)` here.